rabbitmq/README.md

```diff
@@ -169,6 +169,14 @@ RABBITMQ_MANAGEMENT_SSL_KEYFILE
 RABBITMQ_MANAGEMENT_SSL_VERIFY
 ```
 
+If you don't want to store sensitive username and password information in an environment variable, this image also has an option for loading these values from a specified file in the container.  This can be useful when running this image in Docker Swarm for example, which has the ability to mount secrets as files on the container's filesystem.
+
+You can set the following 2 environment variables to point to the location of a file used to set the appropriate RabbitMQ setting.  The file(s) should contain the value that you want to substitute in plaintext.
+```bash
+RABBITMQ_DEFAULT_USER_FILE
+RABBITMQ_DEFAULT_PASS_FILE
+```
+
 ### Setting default user and password
```

Comment:
This new content should probably go at the end of this section.  Also it needs to be put in `content.md` since `README.md` is generated.